app/presenters/publishing_api/working_group_presenter.rb

```diff
@@ -53,7 +53,11 @@ def details
 
     def body
       # It looks 'wrong' using the description as the body, but it isn't
-      Whitehall::GovspeakRenderer.new.govspeak_with_attachments_to_html(item.description, item.attachments, item.email)
+      if item.description.present?
+        Whitehall::GovspeakRenderer.new.govspeak_with_attachments_to_html(item.description, item.attachments, item.email)
```

Comment:
Why wasn't this failing before the upgrade? Or was it failing but we didn't notice?